README.md

```diff
@@ -24,8 +24,17 @@ cd osier
 pip install .
 # to also install the documentation dependencies
 pip install .[doc]
+
+# followed by 
+pip install pyomo==6.4.1
 ```
 
+> [!NOTE]
+> Although `pyomo` is a dependency, the current version of `pyomo` (6.7.1, as of 2/29/24) has a bug
+that prints erroneous errors during an `osier` simulation. Therefore, users are recommended to 
+install a specific version of `pyomo` after the main installation of `osier`. There is an open issue [#50](https://github.com/arfc/osier/issues/50) 
+related to this concern.
```

Comment:
This block is not rendering correctly.
![Screenshot 2024-03-04 at 15-09-33 osier — osier 0 2 1 documentation](https://github.com/arfc/osier/assets/45364492/fe9ed917-d78f-4961-8459-e3fe7a28c531)

Reply:
Because it's a github-flavored markdown block. It renders fine on github. I'm not concerned about it not rendering on the docs. 

If it bothers you, please make an issue and possibly suggest some solutions? I will not be addressing this in the present PR.